package.json

```diff
@@ -37,36 +37,36 @@
     "Ilsun Park <ilsun.park@teradata.com>"
   ],
   "dependencies": {
-    "@angular/animations": "4.3.0",
-    "@angular/common": "4.3.0",
-    "@angular/compiler": "4.3.0",
-    "@angular/core": "4.3.0",
-    "@angular/flex-layout": "2.0.0-beta.8",
+    "@angular/animations": "4.4.3",
+    "@angular/common": "4.4.3",
+    "@angular/compiler": "4.4.3",
+    "@angular/core": "4.4.3",
+    "@angular/flex-layout": "2.0.0-beta.10",
     "@angular/forms": "4.3.0",
     "@angular/http": "4.3.0",
-    "@angular/cdk": "2.0.0-beta.10",
-    "@angular/material": "2.0.0-beta.10",
-    "@angular/platform-browser": "4.3.0",
-    "@angular/platform-browser-dynamic": "4.3.0",
-    "@angular/platform-server": "4.3.0",
-    "@angular/router": "4.3.0",
-    "@covalent/core": "1.0.0-beta.7",
-    "@covalent/dynamic-forms": "1.0.0-beta.7",
-    "@covalent/highlight": "1.0.0-beta.7",
-    "@covalent/http": "1.0.0-beta.7",
-    "@covalent/markdown": "1.0.0-beta.7",
+    "@angular/cdk": "2.0.0-beta.11",
+    "@angular/material": "2.0.0-beta.11",
+    "@angular/platform-browser": "4.4.3",
+    "@angular/platform-browser-dynamic": "4.4.3",
+    "@angular/platform-server": "4.4.3",
+    "@angular/router": "4.4.3",
+    "@covalent/core": "1.0.0-beta.8",
```

Comment:
you can go straight to `1.0.0-beta.8-1` for all the covalent dependencies in this PR